addons/library.xbmc.addon/libXBMC_addon.h

```diff
@@ -47,6 +47,8 @@ typedef intptr_t      ssize_t;
 #define ADDON_HELPER_ARCH       "powerpc-osx"
 #elif defined(__arm__)
 #define ADDON_HELPER_ARCH       "arm-osx"
+#elif defined(__aarch64__)
+#define ADDON_HELPER_ARCH       "arm-osx"
```

Comment:
Yes, it seems strange to me as above is the same and you could have used an or operator there. Would have expected something like "arm-osx64".
Seems like your grouping stuff and I with my limited knowledge of this part of the code don't understand why you might do that.